public/app/listeners/getUserFolder.js

```diff
@@ -0,0 +1,14 @@
+const { GET_USER_FOLDER_CHANNEL } = require('../config/channels');
+const { VAR_FOLDER } = require('../config/config');
+const { ERROR_GENERAL } = require('../config/errors');
+
+const getUserFolder = (mainWindow) => async () => {
+  try {
+    mainWindow.webContents.send(GET_USER_FOLDER_CHANNEL, VAR_FOLDER);
+  } catch (e) {
+    logger.error(e);
+    mainWindow.webContents.send(GET_USER_FOLDER_CHANNEL, ERROR_GENERAL);
+  }
+};
+
+module.exports = getUserFolder;
```

Comment:
Also here, you should add a new line after, as it is good practice to end a file with a new line.